contrib/packages/debian/rules

```diff
@@ -4,8 +4,7 @@
 DEB_PYTHON_SYSTEM := pysupport
 
 include /usr/share/cdbs/1/rules/debhelper.mk
-include /usr/share/cdbs/1/class/python-distutils.mk
 
 clean::
 	rm -rf build build-stamp configure-stamp build/ MANIFEST
-	dh_clean
+	dh_clean
```

Comment:
Do you know why this line is marked as changed?

Reply:
No I don't. :confused:

Reply:
It was removed the new line at the end.